libeventheader-decode-cpp/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ project(eventheader-decode-cpp
     LANGUAGES CXX)
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
+set(CMAKE_CXX_STANDARD 98)  # Ensure projects declare minimum C++ requirement.
```

Comment:
Note: Set a low default for every root project. That forces each subproject to either work with the low default OR correctly declare its requirement.